### PR TITLE
[Mikhail Ilichev] mock 1

### PR DESCRIPTION
**Pull request by miffn3**
https://mikhail-ilichev-task1.firebaseapp.com/

### CODE REVIEW

File: mock1/public/index.html [hunk 1]
```diff
@@ -2,38 +2,124 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Welcome to Firebase Hosting</title>
+    <title>Mock 1</title>
```
Comment (KalachevDev) [inline mock1/public/index.html]: Read about mandatory (or almost mandatory) meta tags and properties and what will happen if you miss it

File: mock1/public/style/main.css [hunk 2]
```diff
@@ -1,75 +1,257 @@
...
+  border-color: #ffffff;
+}
+.wrapper_last-block{
+  padding-bottom: 81px;
+}
+
+.container{
+  box-sizing: border-box;
   display: flex;
+  padding: 65px 7vw;
```
Comment (KalachevDev) [inline mock1/public/style/main.css]: Is it mandatory to use vw here? I see you use pixels everywhere else

File: mock1/public/style/main.css [hunk 3]
```diff
@@ -1,75 +1,257 @@
...
+  min-width: 200px;
+  font-size: 34px;
+  font-weight: 700;
+  text-transform: uppercase;
+  letter-spacing: 8px;
 }
 
-.header__text_color-white{
+.container__image{
+
```
Comment (KalachevDev) [inline mock1/public/style/main.css]: ?

File: mock1/public/style/main.css [hunk 4]
```diff
@@ -1,75 +1,257 @@
...
+.container_intro{
   color: #ffffff;
+  height: 100%;
+}
+
+.container__image_intro{
+  width: 65px;
+  height: 69px;
+  margin-bottom: 5vh;
```
Comment (KalachevDev) [inline mock1/public/style/main.css]: Why not pixels?

File: mock1/public/index.html [hunk 5]
```diff
@@ -1,125 +1,157 @@
...
+      <p class="container__text container__text_intro">
+        We’ve been helping startups launch and grow their businesses.
+        We help you to fill the gap between development,
+        design and product management.
+      </p>
+      <img src="img/blocks/container/Mouse-icon.svg" alt="mouse_icon" class="container__image_mouse-icon" />
+    </div>
+  </section>
+  <section class="wrapper">
+    <div class="container container_border">
```
Comment (KalachevDev) [inline mock1/public/index.html]: bordered I suppose

**Review by KalachevDev — DISMISSED** [review]

**Review by KalachevDev — APPROVED** [review]

**Review by vovcyan — APPROVED** [review]

**Pull request merged**